lib/events.js

```diff
@@ -82,9 +106,10 @@ MapboxEventManager.prototype = {
       return callback();
     }
     var options = this.getRequestOptions(payload);
+    var _this = this;
     this.request(options, function(err,res){
-      if (err) return this.handleError(err, callback);
-      if (res.statusCode != 204) return this.handleError(res, callback);
+      if (err) return _this.handleError(err, callback);
+      if (res.statusCode != 204) return _this.handleError(res, callback);
       if (callback) return callback();
```

Comment:
This is duplicative of line 104 in that `callback` is populated if it is falsy. Line 104 should be removed.